Modules/BookClass.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
+class Book {
```

Comment:
- [ ] This request from the previous reviewer has not been implemented. Disable one-line of code with eslint-disable could be allow at some place. But disable the whole file is what is not allowed. kindly make more research to find how to resolve this. Take a look at below code :point_down: 

```
 class Book { // eslint-disable-line no-unused-vars
     constructor(title, author, id) {
         this.title = title;
         this.author = author;
         this.id = id;
     }
  }
```